docs/source/rabbitmq-tutorial/examples/6-rpc/rpc_client.py

```diff
@@ -12,16 +12,13 @@ class FibonacciRpcClient:
     connection: AbstractConnection
     channel: AbstractChannel
     callback_queue: AbstractQueue
-    loop: asyncio.AbstractEventLoop
 
     def __init__(self) -> None:
         self.futures: MutableMapping[str, asyncio.Future] = {}
-        self.loop = asyncio.get_running_loop()
+        self._loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
```

Comment:
`asyncio.get_event_loop` is deprecated now (https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop).

Reply:
removed loop from init, it's redundant here
we can get it directly in coro